tests/test_yask.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+from sympy import Eq
+
+import pytest  # noqa
+
+from devito import Operator, DenseData, x, y, z
+from devito.dle import YaskGrid
+from devito.parameters import configuration, defaults
+
+pexpect = pytest.importorskip('yask_compiler')  # Run only if YASK is available
```

Comment:
I take it this needs upstream changes before we can set it live in travis? Just curious...